internal/charmstore/server.go

```diff
@@ -46,6 +47,10 @@ type ServerParams struct {
 	// authentication.
 	AgentUsername string
 	AgentKey      *bakery.KeyPair
+
+	// StatsCacheMaxAge is the maximum length of time between
+	// refreshes of entities in the stats cache.
+	StatsCacheMaxAge time.Duration
```

Comment:
// If it is zero, it defaults to 1 hour.

... although, perhaps if it's zero the cache should be disabled?